tests/PHPUnit/System/expected/test_OneVisitorTwoVisits_showColumnsWithProcessedMetrics___API.getProcessedReport_day.xml

```diff
@@ -229,5 +229,6 @@
 	</reportMetadata>
 	<reportTotal>
 		<nb_visits>2</nb_visits>
+		<revenue>43</revenue>
```

Comment:
Here an example where one can see that it fixes report total calculation as mentioned before.